client/task-list/tasks/payments/recommended-ribbon.js

```diff
@@ -0,0 +1,22 @@
+/**
```

Comment:
Just FYI, @elizaan36 has [mentioned the possibility of getting rid of the ribbon](https://github.com/woocommerce/woocommerce-admin/issues/6683#issuecomment-816924780), although we don't need to worry about that here.